csharp/Platform.Data.Doublets.Sequences/Unicode/UnicodeSequenceToStringConverter.cs

```diff
@@ -46,11 +49,13 @@ public class UnicodeSequenceToStringConverter<TLinkAddress> : LinksOperatorBase<
         /// <para></para>
         /// </param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public UnicodeSequenceToStringConverter(ILinks<TLinkAddress> links, ICriterionMatcher<TLinkAddress> unicodeSequenceCriterionMatcher, ISequenceWalker<TLinkAddress> sequenceWalker, IConverter<TLinkAddress, char> unicodeSymbolToCharConverter) : base(links)
+        public UnicodeSequenceToStringConverter(ILinks<TLinkAddress> links, ICriterionMatcher<TLinkAddress> unicodeSequenceCriterionMatcher, ISequenceWalker<TLinkAddress> sequenceWalker, IConverter<TLinkAddress, char> unicodeSymbolToCharConverter, TLinkAddress unicodeSequenceMarker) : base(links)
         {
             _unicodeSequenceCriterionMatcher = unicodeSequenceCriterionMatcher;
             _sequenceWalker = sequenceWalker;
             _unicodeSymbolToCharConverter = unicodeSymbolToCharConverter;
+            _unicodeSequenceMarker = unicodeSequenceMarker;
+
         }
 
```

Comment:
Because we have both `links` and `unicodeSequenceMarker`, we can create constructor overload without `ICriterionMatcher<TLinkAddress> unicodeSequenceCriterionMatcher`.
```cs
public UnicodeSequenceToStringConverter(ILinks<TLinkAddress> links, ISequenceWalker<TLinkAddress> sequenceWalker, IConverter<TLinkAddress, char> unicodeSymbolToCharConverter, TLinkAddress unicodeSequenceMarker)
```
Both `links` and `unicodeSequenceMarker` could be passed to the constructor of `unicodeSequenceCriterionMatcher`.
Adding such constructor overload will allow users of this class to pass less arguments, and get work done faster.

Reply:
https://github.com/linksplatform/Data.Doublets.Sequences/issues/20#issuecomment-1069296749